authorization/v2/uniformResourceIdentifier.go

```diff
@@ -24,6 +24,64 @@ func NewUriFromRequest(resource resources.Resource, handler gin.HandlerFunc, ctx
 	}
 }
 
+// NewURIFromString parses the string representation of a URI into the URI struct.
```

Comment:
We should probably use the full name of the struct in the comment:
`// NewURIFromString parses the string representation of a URI into the UniformResourceIdentifier struct.`